src/main/java/edu/kit/ipd/crowdcontrol/objectservice/feedback/FeedbackException.java

```diff
@@ -0,0 +1,13 @@
+package edu.kit.ipd.crowdcontrol.objectservice.feedback;
+
+import edu.kit.ipd.crowdcontrol.objectservice.database.model.tables.Experiment;
+
+/**
+ * Gets thrown, if a
+ */
+public class FeedbackException extends Exception {
+
+    public FeedbackException(String message) {
```

Comment:
What happens if you just skip this constructor, calling super(...) should not need to create a indidual constructor.